data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/unapproachable_east/una_classes.lst

```diff
@@ -2,7 +2,7 @@
 # these are 35 skills/spell references only
 
 # Class Name			Hit Dice	Type			Abbreviation	Source Page		Combat bonus										Bonus
-CLASS:Aglarondan Griffonrider	HD:8		TYPE:PC.Prestige	FACT:Abb|Gri		SOURCEPAGE:p.18	BONUS:COMBAT|BASEAB|classlevel("APPLIEDAS=NONEPIC")|TYPE=Base.REPLACE|PREVAREQ:UseAlternateBABProgression,0	BONUS:SAVE|BASE.Fortitude,BASE.Will|classlevel("APPLIEDAS=NONEPIC")/3	BONUS:SAVE|BASE.Reflex|classlevel("APPLIEDAS=NONEPIC")/2+2
+CLASS:Aglarondan Griffonrider	HD:8		TYPE:PC.Prestige	CLASSTYPE:Prestige		FACT:Abb|Gri		SOURCEPAGE:p.18	BONUS:COMBAT|BASEAB|classlevel("APPLIEDAS=NONEPIC")|TYPE=Base.REPLACE|PREVAREQ:UseAlternateBABProgression,0	BONUS:SAVE|BASE.Fortitude,BASE.Will|classlevel("APPLIEDAS=NONEPIC")/3	BONUS:SAVE|BASE.Reflex|classlevel("APPLIEDAS=NONEPIC")/2+2
```

Comment:
For 6.06 the tag 'CLASSTYPE:x' is replaced by 'FACT:ClassType|x'

Load this up and it will have deprecated warnings.